openshift-client-python/openshift-client-python.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: openshift-client-python
+spec:
+  workspaces:
+    - name: kubeconfig-mount
+      mountPath: /tekton/home/.kube
+  params:
+    - name: SCRIPT
+      description: Python script to be executed
+      type: string
+      default: |
+        #!/usr/bin/python
+
+        import openshift as oc
+
+        print('OpenShift client version: {}'.format(oc.get_client_version()))
+        print('OpenShift server version: {}'.format(oc.get_server_version()))
+  steps:
+    - name: execute-script
+      image: quay.io/vinamra2807/openshift-python-runner:latest
```

Comment:
What this image is doing Do we have any Dockerfile reference for the same

Reply:
provided in the PR description

Reply:
The Dockerfile consists of the oc and the python binary

Reply:
Could we copy the Dockerfile that makes this image into this directory, so that users can build and use their own if they want to?

Reply:
Sure...will do that